webapp/app.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+
+from webapp.config import Config
+
+CONFIG = Config()
+
+
+def create_app() -> Flask:
+    app = Flask(__name__)
+    CONFIG.check_required_env_vars()
+    app.config.from_object(CONFIG)
+
+    @app.route("/")
+    def hello_world() -> str:
+        return f"Hello, World! WORKSPACE={app.config['WORKSPACE']}"
```

Comment:
While we can access additional app configs directly from the `webapp.config.Config` object, I do think it would be helpful if __all__ configs are stored in and accessed from a single, central location, and given that this is a Flask app, it feels like `app.config` is most appropriate. 

Reading the Flask docs: 
> Independent of how you load your config, there is a config object available which holds the loaded configuration values: The [config](https://flask.palletsprojects.com/en/3.0.x/api/#flask.Flask.config) attribute of the [Flask](https://flask.palletsprojects.com/en/3.0.x/api/#flask.Flask) object. This is the place where Flask itself puts certain configuration values and also where extensions can put their configuration values. **But this is also where you can have your own configuration.**

From my perspective, I think we should prioritize accessing configurations from `app.config` when needed and only directly access `webapp.config.Config` if absolutely necessary...

@ehanson8 @ghukill What do you think? 🤔

Reply:
Agreed.

If we find ourselves in a situation where there is code that a) might need to access configurations, but b) is _not_ expected to get invoked in a flask app context (seems unlikely), then we can always access directly from the `CONFIG` object.

Reply:
I totally agree on centralizing and that `app.config` is the most appropriate for a Flask app unless there's a very good reason for `webapp.config.Config`